OpenEdXMobile/res/layout/fragment_whats_new_item.xml

```diff
@@ -1,63 +1,53 @@
-                android:layout_gravity="bottom"
-                android:paddingStart="@dimen/whats_new_image_horizontal_margin"
-                android:paddingEnd="@dimen/whats_new_image_horizontal_margin"
-                tools:ignore="ContentDescription"
-                tools:src="@drawable/screen_1" />
-        </FrameLayout>
+        android:layout_height="0dp"
+        android:layout_margin="@dimen/edx_half_margin"
+        app:layout_constraintDimensionRatio="1:1.25"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        tools:ignore="ContentDescription"
+        tools:src="@drawable/screen_1" />
 
-        <LinearLayout
-            android:id="@+id/ll_message_container"
-            android:layout_width="match_parent"
-            android:layout_height="0dp"
-            android:layout_weight="0.3"
-            android:background="@color/primaryBaseColor"
-            android:gravity="top|center_horizontal"
-            android:orientation="vertical"
-            android:paddingStart="@dimen/edx_margin"
-            android:paddingTop="@dimen/edx_margin"
-            android:paddingEnd="@dimen/edx_margin">
+    <LinearLayout
+        android:id="@+id/ll_message_container"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="@dimen/edx_default_margin"
+        android:orientation="vertical"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/image">
 
-            <TextView
-                android:id="@+id/title"
-                style="@style/semibold_text"
-                android:layout_width="wrap_content"
-                android:layout_height="wrap_content"
-                android:gravity="center"
-                android:textColor="@color/neutralWhite"
-                android:textSize="@dimen/edx_base"
-                tools:text="App Feedback" />
+        <TextView
+            android:id="@+id/title"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_gravity="center_horizontal"
+            android:fontFamily="?attr/fontBold"
+            android:textColor="@color/neutralWhite"
+            android:textSize="@dimen/edx_x_large"
+            tools:text="App Feedback" />
 
-            <TextView
-                android:id="@+id/message"
-                android:layout_width="wrap_content"
-                android:layout_height="wrap_content"
-                android:gravity="center"
-                android:paddingTop="@dimen/widget_margin"
-                android:scrollbars="vertical"
-                android:textColor="@color/neutralWhite"
-                android:textSize="@dimen/edx_base"
-                tools:text="This is the description." />
-        </LinearLayout>
+        <TextView
+            android:id="@+id/message"
```

Comment:
The message is getting trimmed on the small screens. Tested on Nexus 5. 
(Screenshot shared on slack)

Reply:
Some minor changes in font size and other dimensions to fit the designs on small resolutions devices.